packages/braintree-integration/src/braintree.ts

```diff
@@ -459,6 +460,38 @@ export interface BraintreeVenmoCreatorConfig extends BraintreeModuleCreatorConfi
 //     createInitOptions(options: Partial<VisaCheckoutInitOptions>): VisaCheckoutInitOptions;
 // }
 
+/**
+ *
+ * Braintree US Bank Account
```

Comment:
Do we need to document `US` here? Is there a possibility that this might be same for a non US account?

Reply:
@animesh1987 it is only for US account

Reply:
Thanks, just wondering if there is a possibility that we have this available for non US accounts. Then the name might outlive it's purpose. Can we rename it in a more generic way and not tie an interface to a country code?